src/app/services/auth.service.ts

```diff
@@ -10,13 +10,15 @@ export class AuthService {
 
   constructor(private http: HttpClient) { }
 
+  /**
+   * Logs in a registered user
+   *
+   * @param {object} userInfo contains all the users information
+   *
+   * @return {observable} Observable containing the user's info
+   */
   login(userInfo) {
       return this.http.post(`${this.apiBaseUrl}/v1/login`, userInfo)
       .map((response: {data}) => response)
```

Comment:
this line adds the additional performance cost of the `map` operation. it would be negligible for small requests but if you need to scale then every little counts